core/src/main/groovy/noe/rhel/server/workspace/WorkspaceHttpdTomcatsBaseOS.groovy

```diff
@@ -46,22 +47,42 @@ class WorkspaceHttpdTomcatsBaseOS extends WorkspaceMultipleTomcats{
     httpd.updateConfSetBindAddress(hostIpAddress)
 
     copyModulesIfMissing(httpd)
+    copyConfIfMissing(httpd)
     workspaceTomcat.prepare(true) //always true due to running with JWS 5 `ewsVersion >= new Version('3.1.0-DR0')`
 
     createAdditionalTomcatsWithRegistrations(numberOfAdditionalTomcats)
     originalTomcatHosts.putAll(originalTomcatHostsIpAddresses())
   }
 
   private void copyModulesIfMissing(Httpd httpd) {
```

Comment:
Comment this method would be beneficial, goal of it seems non standard.

Reply:
Commented, present in next commit

Reply:
see my comments to doc for configurations